src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const dataSpans = document.querySelectorAll('.population');
+const totalElement = document.querySelector('.total');
+const averageElement = document.querySelector('.average');
+
+function removeCommas(str) {
+  return str.replace(/,/g, '');
+}
+
+function formatNumberWithCommas(number) {
```

Comment:
That's nice you've implemented your own solution 👍 
So just FYI, there is a method that does the same (with the correct locale, but the default locale works as expected): [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString)